chia/server/node_discovery.py

```diff
@@ -62,7 +63,7 @@ def __init__(
         random.shuffle(dns_servers)  # Don't always start with the same DNS server
         if introducer_info is not None:
             self.introducer_info: Optional[PeerInfo] = PeerInfo(
-                introducer_info["host"],
+                str(get_host_addr(introducer_info["host"], prefer_ipv6=False)),
```

Comment:
is it OK to make a blocking call here? I assume `get_host_addr()` is blocking in case it needs to perform a DNS lookup. If it's OK, I think it warrants a comment in the code to make it clear to readers

Reply:
See 5009987591390d12aee466339e5d30e02709bcae.